packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -446,6 +463,7 @@ describe('internal-plugin-metrics', () => {
 
         assert.calledWith(getIdentifiersSpy, {
           correlationId: 'correlationId',
+          preLoginId: undefined
```

Comment:
issue: also need a test where preLoginId is defined

Reply:
there is one already